app/dashboard/_components/RiskAndOpps/RiskOppCard.tsx

```diff
@@ -0,0 +1,23 @@
+import Card from '@/components/Card'
+import { risksData } from '@/app/dashboard/_components/RiskAndOpps/data/RisksData'
```

Comment:
i would pass the data as props from the 'uppermost' component aka EntireSection, that way this card is more reusable